### PR TITLE
refactored magic strings and took them out to consts

### PR DESCRIPTION
**Pull request by IlyaBelik**
Refactored 'magic strings and number' from Models and Modules(1-12)
Created new consts files for saving strings and numbers
image

### CODE REVIEW

File: consts/db-refs.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  CONSTRUCTOR_FRONT_POCKET: 'constructorFrontPocket',
+  PATTERN: 'Pattern',
+  USER: 'User',
+  COLOR: 'Color',
+  MATERIAL: 'Material',
+};
+
+module.exports = {
+  DB_REFS,
+};
```
Comment (AndriyLymych) [inline consts/db-refs.js:16]: it is usles because we can get this data from db-collections-name

File: consts/db-tables-names.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+const DB_TABLES_NAMES = {
```
Comment (AndriyLymych) [inline consts/db-tables-names.js:1]: call this const `DB_COLLECTIONS_NAME`

File: consts/delivery-services.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+const SERVICES = {
```
Comment (AndriyLymych) [inline consts/delivery-services.js:1]: call it more precisely

File: consts/sizes.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+const SIZES = {
```
Comment (AndriyLymych) [inline consts/sizes.js:1]: sizes of what?call it more precisely

File: consts/currency.js [hunk 5]
```diff
@@ -3,6 +3,16 @@ const CURRENCY = {
...
 };
 
+const LOCALES = {
+  UK: 'uk',
+};
+
+const COUNTS = {
+  COUNT: 'count',
+};
+
```
Comment (AndriyLymych) [inline consts/currency.js:13]: that consts are not about currency

File: consts/payments.js [hunk 6]
```diff
@@ -0,0 +1,7 @@
+const PAYMENT_METHOD = {
+  CASH: 'Cash',
```
Comment (AndriyLymych) [inline consts/payments.js:2]: write'CASH' , not 'Cash'

File: consts/statuses.js [hunk 7]
```diff
@@ -0,0 +1,9 @@
+const STATUSES = {
```
Comment (AndriyLymych) [inline consts/statuses.js:1]: statuses of what?

File: modules/closures/closures.model.js [hunk 8]
```diff
@@ -13,4 +17,4 @@ const closureSchema = new mongoose.Schema({
   available: Boolean,
 });
 
-module.exports = mongoose.model('Closure', closureSchema);
+module.exports = mongoose.model(CLOSURE, closureSchema);
```
Comment (AndriyLymych) [inline modules/closures/closures.model.js:20]: replace magic string in line 13

File: package-lock.json [hunk 9]
```diff
@@ -3559,25 +3559,6 @@
...
-          "resolved": "https://registry.npmjs.org/graphql-upload/-/graphql-upload-11.0.0.tgz",
-          "integrity": "sha512-zsrDtu5gCbQFDWsNa5bMB4nf1LpKX9KDgh+f8oL1288ijV4RxeckhVozAjqjXAfRpxOHD1xOESsh6zq8SjdgjA==",
-          "requires": {
-            "busboy": "^0.3.1",
-            "fs-capacitor": "^6.1.0",
-            "http-errors": "^1.7.3",
-            "isobject": "^4.0.0",
-            "object-path": "^0.11.4"
-          }
-        }
```
Comment (AndriyLymych) [inline package-lock.json:3580]: dont remove this)

File: package-lock.json [hunk 10]
```diff
@@ -8065,6 +8051,17 @@
...
+      "requires": {
+        "busboy": "^0.3.1",
+        "fs-capacitor": "^2.0.4",
+        "http-errors": "^1.7.3",
+        "object-path": "^0.11.4"
+      }
+    },
     "graphql-ws": {
       "version": "4.1.0",
       "resolved": "https://registry.npmjs.org/graphql-ws/-/graphql-ws-4.1.0.tgz",
```
Comment (AndriyLymych) [inline package-lock.json:8067]: dont touch package-lock

File: consts/delivery-services.js [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+const DELIVERY_SERVICES_METHODS = {
+  GET_CITIES: 'getCities',
+  GET_STREET: 'getStreet',
+  GET_WAREHOUSES: 'getWarehouses',
+  GET_COUNTER_PARTY_CONTACT_PERSONS: 'getCounterpartyContactPersons',
+  GET_COUNTER_PARTIES: 'getCounterparties',
+  GET_DOCUMENT_PRICE: 'getDocumentPrice',
+  SAVE: 'save',
+  POST: 'POST',
+  GET: 'GET',
```
Comment (AndriyLymych) [inline consts/delivery-services.js:22]: create file REQUESTS_METHODS and put POST and GET into this file

**Comment by IlyaBelik** [comment]
done

File: consts/currency.js [hunk 12]
```diff
@@ -3,6 +3,11 @@ const CURRENCY = {
   USD: 'USD',
 };
 
+const LOCALES = {
+  UK: 'uk',
+};
+
```
Comment (AndriyLymych) [inline consts/currency.js:9]: LOCALES its not about currency too

File: consts/db-collections-names.js [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+  MODEL: 'Model',
+  SIZE: 'Size',
+  CONSTRUCTOR_BASICS: 'constructorBasics',
+  CONSTRUCTOR_BOTTOM: 'constructorBottom',
+  CONSTRUCTOR_FRONT_POCKET: 'constructorFrontPocket',
+  PATTERN: 'Pattern',
+  USER: 'User',
+  COLOR: 'Color',
+  MATERIAL: 'Material',
+};
```
Comment (AndriyLymych) [inline consts/db-collections-names.js:34]: `DB_REFS` its the same as `DB_COLLECTIONS_NAME`

**Comment by IlyaBelik** [comment]
joined db-refs with db-collections

**Review by AndriyLymych — APPROVED** [review]

**Review by Haytham1337 — APPROVED** [review]
Well Done!

**Review by Kiyooooko — APPROVED** [review]

**Pull request merged**